invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/RemoveFromCommunity/RemoveFromCommunityAction.js

```diff
@@ -151,7 +151,7 @@ export class RemoveFromCommunityAction extends Component {
             {error && (
               <ErrorMessage
                 header={i18next.t("Something went wrong")}
-                content={error.response?.data?.errors[0]?.message || error.message}
+                content={error.response?.data?.errors?.[0]?.message || error.response?.data?.message || error.message}
```

Comment:
ErrorMessage component should already be able to handle different types of errors, please check if thats the case, if yes, then you can simply pass an error object and the component will parse it